docs: align docs with `FeedbackDataset` refactor using `tab-set` (#3531)
recognai/rubrix: docs/_source/guides/llms/conceptual_guides/rm.md

# Add records to the dataset
dataset.add_records(records)
```

::::{tab-set}

:::{tab-item} Argilla 1.14.0 or higher
```python

> # This publishes the dataset with the records into Argilla and returns the dataset in Argilla

Author: ```suggestion
# This publishes the dataset with its records to Argilla and returns the dataset in Argilla
```

> remote_dataset = dataset.push_to_argilla(name="my-dataset", workspace="my-workspace")

Reviewer: The variable `remote_dataset` is not used in the rest of the code snippets in the page, so maybe `dataset.push_to_argilla()` is enough in this case?

Author: Even though it may not be used later, I think it's nice to show that they need to capture the return object, as the `warning` will be removed in the next release, and that affects users if they copy-paste the snippets pretending to use those further

Reviewer 2: I agree with Alvaro. Even if it's not used, it's good to have it there so the user knows that the methods returns something that can be used.

```
:::

:::{tab-item} Lower than Argilla 1.14.0
```python

> # This publishes the dataset with the records into Argilla and turns the dataset objetct into a dataset in Argilla

Reviewer 2: ```suggestion
# This publishes the dataset with its records to Argilla and returns the dataset in Argilla
```

> # This publishes the dataset with the records into Argilla and turns the dataset objetct into a dataset in Argilla

Reviewer 2: ```suggestion
# This publishes the dataset with its records to Argilla and turns the dataset object into a dataset in Argilla
```
